python-stdlib/unittest/unittest.py

```diff
@@ -188,8 +188,14 @@ def wasSuccessful(self):
 # TODO: Uncompliant
 def run_class(c, test_result):
     o = c()
+    set_up_class = getattr(o, "setUpClass", lambda: None)
+    tear_down_class = getattr(o, "tearDownClass", lambda: None)
     set_up = getattr(o, "setUp", lambda: None)
     tear_down = getattr(o, "tearDown", lambda: None)
+
+    if set_up_class:
+        set_up_class()
```

Comment:
In the `getattr` line above, if the function doesn't exist it defaults to `lambda: None`. This is done so that you don't need the `if set_up_class` here, you can just run `set_up_class()` and if it doesn't exist it'll just run the no-op lambda function

Reply:
Thanks for the suggestion, fixed.